apps/frontend/src/components/LandingPage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Box, Button, Flex, Image, VStack } from "@chakra-ui/react";
+
+function AuthButtons() {
+  /* Temporary styles for buttons; will likely be replaced with
+  Chakra's 'useStyleConfig'. */
+  const buttonStyles = {
+    width: "50%",
+    height: "4em",
+  };
+
+  /* Using VStack right now; this is similar to Flex; may decide to
+  switch to Flex based on review. */
+  return (
+    <VStack w={"100%"} p={"10px 0"}>
```

Comment:
Using fixed ```px``` might not be the best choice if we want responsive design

Reply:
Fixed.